base/Dockerfile

```diff
@@ -26,6 +26,7 @@ ENV USER_ID=1001
 RUN mkdir -p ${APP_ROOT}
 RUN useradd -u ${USER_ID} -r -g 0 \
             -s /sbin/nologin \
-            -c "Default Application User" default
+            -c "Default LFH Application User" \
+            lfh
```

Comment:
I changed our app user from "default" to "lfh".
Once this PR is merged I'll update the LFH connect image and will create a new tag.